src/bundle.c

```diff
@@ -2502,6 +2502,41 @@ gboolean load_manifest_from_bundle(RaucBundle *bundle, RaucManifest **manifest,
 	return res;
 }
 
+static gboolean read_complete_dm_device(gchar *dev, GError **error)
+{
+	int checkfd = -1;
+	char checkbuf[65536];
+	int bytesRead = sizeof(checkbuf);
+	gboolean ret = TRUE;
+
+	checkfd = g_open(dev, O_RDONLY | O_CLOEXEC, 0);
+	if (checkfd < 0) {
+		int err = errno;
+		g_set_error(error,
+				G_FILE_ERROR,
+				g_file_error_from_errno(err),
+				"Failed to open %s: %s", dev, g_strerror(err));
+		return FALSE;
+	}
+
+	while (ret && bytesRead > 0)
+	{
```

Comment:
Please use our existing style. You can use `./uncrustify.sh` to fix this automatically.

Reply:
Uncrustify did not automatically fix this formatting. I have extended the uncrustify settings to catch theses.